fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -220,6 +222,39 @@ then
 	log_Err_Abort "the --usejacobian option must be 'true' or 'false'"
 fi
 
+
+# ------------------------------------------------------------------------------
+#  Legacy Style Data Options
+# ------------------------------------------------------------------------------
+
+UseMask=`opts_GetOpt1 "--usemask" $@`                                   # what mask to use for the final bold (T1_fMRI_FOV combination of T1w and FOV masks: the default, T1: T1w image based mask, DILATEDT1: dilated T1w image based mask, NONE: none)
```

Comment:
`--usemask` seems more in style with one of our TRUE/FALSE/YES/NO arguments.  Would `--maskmode` or `--maskstyle` be a better choice?

Reply:
I'll change it to `--boldmask` as that's what it is mask for BOLD.